internal/distro/rhel8/sap.go

```diff
@@ -159,7 +159,7 @@ func SapPackageSet(t *imageType) rpmmd.PackageSet {
 		},
 	}
 
-	if t.arch.distro.osVersion == "8.4" {
+	if t.arch.distro.osVersion == "8.4" || t.arch.distro.osVersion == "8.5" {
```

Comment:
You could use https://github.com/osbuild/osbuild-composer/blob/f6bd62e8388bcb5a9087525222dbf69827431238/internal/common/distro.go#L61 instead. The reason for 8.4 being the first specified release is that SAP images were not built by composer prior to 8.4.